flink/v1.14/flink/src/test/java/org/apache/iceberg/flink/source/TestStreamingMonitorFunction.java

```diff
@@ -226,7 +226,7 @@ private void writeRecords(List<Record> records) throws IOException {
   }
 
   private StreamingMonitorFunction createFunction(ScanContext scanContext) {
-    return new StreamingMonitorFunction(TestTableLoader.of(tableDir.getAbsolutePath()), scanContext);
+    return new StreamingMonitorFunction(TestTableLoader.of(tableDir.getAbsolutePath()), scanContext, false);
```

Comment:
Where is this called from? Is it okay to do this or should locality be part of the scan context?

Reply:
I think it's a good idea to make it part of the Scan context.

Reply:
I've tried in the second commit, please help to review.Thanks.